src/instructlab/lab.py

```diff
@@ -885,6 +885,14 @@ def convert(self, value, param, ctx) -> "torch.device":
     show_default=True,
     help="model name to use in training",
 )
+@click.option(
+    "--batch-size",
+    "batch_size",
+    default=1,
+    help=(
+        "Batch size for training. Increases GPU usage."
+    )
+)
```

Comment:
It looks like this is Linux-only? If so, could you update the text to indicate that?

Also, if you think about a user trying to use the tool, how should they decide if they should use this option? Is there text you could add to give any guidance?

Maybe the guidance should go in here?  https://github.com/instructlab/instructlab/blob/main/docs/gpu-acceleration.md